### DISCUSSION TITLE
Are user inputs escaped automatically?

### DISCUSSION THREAD

**Discussion by wuifdesign** [emotion-js/emotion#2681] [Q&A]
Will the JS be evaluated when using the `ThemeProvider` or any other way using user input like:

```
<Element color={someUserVariable} />

const Element = styled.div(({ color }) => ({
  color: color
}))
```

`styled-components` has a separate info about it (https://styled-components.com/docs/advanced#security) but i can't find anything about emotion.

**Comment by Andarist** [comment]
We don't do anything special on our end here, so the info in the Styled Components docs applies to Emotion as well.
